Pd 595 add table glyphs to glyphs sketch library

audroue opened the pull request.

audroue committed:
Added Table component glyphs

- Select input glyph
- Unsorted glyph
- Sort ascending glyph
- Sort descending glyph
- Settings ellipsis glyph

audroue committed:
Save on cover page

DesignerDave commented:
Hey!

Just a note on naming – we should name a few of these glyphs more generically.

The ones that stuck out to me are:
- `Settings Ellipsis` -> `Vertical Ellipsis`
- `Select Input` -> Maybe `Up Down Carets`

audroue committed:
Renamed Glyphs

- Settings Ellipsis to Vertical Ellipsis
- Select Input to Maybe Up Down Carets

audroue commented:
Makes sense! I renamed them.

DesignerDave reviewed (approved):
LGTM!
